logging/cluster-logging-external.adoc

```diff
@@ -41,7 +41,50 @@ include::modules/cluster-logging-collector-log-forward-syslog.adoc[leveloffset=+
 
 include::modules/cluster-logging-collector-log-forward-cloudwatch.adoc[leveloffset=+1]
 
+=== Forwarding logs to Amazon CloudWatch from STS enabled clusters
+For clusters with STS enabled, you can create a secret using the xref:../authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc[Cloud Credential Operator(CCO)] utility `ccoctl`. You can use the CCO utility `ccoctl` to create the required AWS resources xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#cco-ccoctl-creating-individually_cco-mode-sts[individually], or xref:../authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc#cco-ccoctl-creating-at-once_cco-mode-sts[with a single command]. 
+
```

Comment:
These instructions are not for logging, but rather for setting up an sts-enabled cluster. 
Once a cluster is set up, they will either need to:
Create the policy and role at AWS manually, and then also create the secret manually (shown in a few parts of https://mobb.ninja/docs/rosa/sts-cluster-logging-addon/)        
OR
Create a credential request specific to logging.   Once created, the customer can then use the utility to apply the CR using the `ccoctl aws create-iam-roles` command, which will create the new role and policy at AWS, and place your secret spec in the output directory..  They will then need to apply the secret, created by the utility's output directory.

Reply:
https://source.redhat.com/personal_blogs/running_pods_in_openshift_with_aws_iam_roles_for_service_accounts
The top part  "Prepare IAM roles and link with Openshift service accounts - Option 1: using ccoctl tool" is fairly accurate on how to use the cco utility to create the necessary role and secret and other assets.    

This script snippet gives an example of a CR for cloudwatch:  https://github.com/openshift/cluster-logging-operator/blob/bccc0149ba35e606433a98133bb4e47e6c460923/hack/aws-sts-cluster-install.sh#L237-L260

The commands below that (beginning line:264) also show the next steps required of creating the role (via the policy in the CR), then applying the secret (which is a yaml file created in the output directory).